src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicStream.cs

```diff
@@ -954,21 +894,6 @@ private static uint HandleEventStartComplete(State state, ref StreamEvent evt)
 
         private static uint HandleEventSendShutdownComplete(State state, ref StreamEvent evt)
```

Comment:
I believe the whole `HandleEventSendShutdownComplete` can be removed, we already return `Success` in case of skipped events in `HandleEvent`
https://github.com/dotnet/runtime/blob/ca3b27963931172f0535fff644e1fae23f406193/src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicStream.cs#L832-L833

Reply:
Done